### ISSUE TITLE
Ability to compare plans

### ISSUE THREAD

**Issue by KES777** [dalibo/pev2#377]
For example I have two plans:
https://explain.dalibo.com/plan/faD https://explain.dalibo.com/plan/UNl

Would be nice to compare nodes of both and highlight by red where differences

**Comment by gleu** [comment]
Agreed. Comparing durations/buffers/etc would have no sense to me, but comparing node types would be great (for example, I'd like to quickly find if the new plan uses a different join or a different scan). It might also be interesting to compare costs, and estimated rows.
